### PR TITLE
bretts buttony commit. 

### PR DESCRIPTION
**Pull request by breadbullock**
Separated out the buttons and gave props to the components to alter their state of mind.

Check it outtttttttttt

### CODE REVIEW

File: cutten-web/src/components/AnalyzeButton.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isClicked: false,
+    }
+  }
+
+  onClick = () => {
+    this.setState({isClicked: true})
```
Comment (adamaho) [inline cutten-web/src/components/AnalyzeButton.tsx]: Why do we care if this button has been clicked? get rid of that bretts a dev thing

File: cutten-web/src/components/SelectButton.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+import {
+  Button
+} from 'kingsbury/lib';
+
+interface IUploadButtonState {
+  file: File | null;
+}
+
+class SelectButton extends React.Component<any, IUploadButtonState> {
```
Comment (adamaho) [inline cutten-web/src/components/SelectButton.tsx]: Can we call this `SelectFileButton` ?

File: cutten-web/src/components/UploadButtonV2.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  file: File | null;
+}
+
+export const UPLOAD_FILE = gql`
+  mutation fuckyouu($file: Upload!) {
+    upload(file: $file)
+  }
+`;
+
+class UploadButtonV2 extends React.Component<any, IUploadButtonState> {
```
Comment (adamaho) [inline cutten-web/src/components/UploadButtonV2.tsx]: Call this `UploadButton` if you are getting rid of the original one.

File: cutten-web/src/components/Main.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      this.setState({
+        file: files[0]
+      });
+    }
+  }
+
+  render() {
+    let { file } = this.state;
+
+    return (
```
Comment (adamaho) [inline cutten-web/src/components/Main.tsx]: We just want to show 3 buttons in a row? what happens when I click Upload or Analyze when there is no file?

File: cutten-server/src/api/upload.resolvers.ts [hunk 5]
```diff
@@ -23,6 +23,7 @@ export default {
       // read the file and write file to disk
       await readStream.pipe(writeStream);
 
+      console.log("how are ya nowwwwww")
```
Comment (adamaho) [inline cutten-server/src/api/upload.resolvers.ts]: get this shit out of here.

File: cutten-web/src/components/Main.tsx (hunk 4 above)
Comment (breadbullock) [reply cutten-web/src/components/Main.tsx]: You can't click them, they're disabled

File: cutten-web/src/components/SelectButton.tsx (hunk 2 above)
Comment (breadbullock) [reply cutten-web/src/components/SelectButton.tsx]: 10-4 chief

File: cutten-web/src/components/UploadButtonV2.tsx (hunk 3 above)
Comment (breadbullock) [reply cutten-web/src/components/UploadButtonV2.tsx]: Roger, roger

**Pull request merged**